osbs/conf.py

```diff
@@ -272,6 +272,9 @@ def get_registry_api_versions(self, platform=None):
                                 "registry_api_versions",
                                 default='v1,v2')
         versions = value.split(',')
+        if not platform:
```

Comment:
How about `if platform is None`, to do an explicit check against the default value for that parameter?